tests/test_decorator_appender.py

```diff
@@ -1,26 +1,55 @@
 import pathlib
```

Comment:
Rename the python file to test_strat.py .

Reply:
Good point

Reply:
Updated and moved into the testing folder for fetching